### ISSUE TITLE
Error executing process > 'pVACseq (sample1)'

### ISSUE THREAD

**Issue by gri11** [icbi-lab/nextNEOpi#60]
```
Error executing process > 'pVACseq (sample1)'

Caused by:
  Process `pVACseq (sample1)` terminated with an error exit status (1)

Command executed:

  pvacseq run \
      --iedb-install-directory /opt/iedb \
      -t 10 \
      -p sample1_vep_phased.vcf.gz \
      -e1 8,9,10,11 \
      -e2 15,16,17,18,19,20,21,22,23,24,25 \
      --normal-sample-name sample1_normal \
      --tumor-purity 0.42 \
       \
      --netmhc-stab \
      --binding-threshold 500 --top-score-metric median --minimum-fold-change 0.0 --normal-cov 5 --tdna-cov 10 --trna-cov 10 --normal-vaf 0.02 --tdna-vaf 0.25 --trna-vaf 0.25 --expn-val 1 --maximum-transcript-support-level 1 \
      sample1_hc_vep_pick.vcf.gz sample1_tumor HLA-C*14:02 NetMHCpan NetMHCpanEL MHCflurry MHCflurryEL NetMHCIIpan NetMHCIIpanEL ./
  
  if [ -e ./MHC_Class_I/sample1_tumor.filtered.tsv ]; then
      mv ./MHC_Class_I/sample1_tumor.filtered.tsv ./MHC_Class_I/sample1_tumor_HLA-C*14:02.filtered.tsv
  fi
  if [ -e ./MHC_Class_I/sample1_tumor.all_epitopes.tsv ]; then
      mv ./MHC_Class_I/sample1_tumor.all_epitopes.tsv ./MHC_Class_I/sample1_tumor_HLA-C*14:02.all_epitopes.tsv
  fi
  if [ -e ./MHC_Class_II/sample1_tumor.filtered.tsv ]; then
      mv ./MHC_Class_II/sample1_tumor.filtered.tsv ./MHC_Class_II/sample1_tumor_HLA-C*14:02.filtered.tsv
  fi
  if [ -e ./MHC_Class_II/sample1_tumor.all_epitopes.tsv ]; then
      mv ./MHC_Class_II/sample1_tumor.all_epitopes.tsv ./MHC_Class_II/sample1_tumor_HLA-C*14:02.all_epitopes.tsv
  fi

Command exit status:
  1

Command output:
  Executing MHC Class I predictions
  Converting .vcf to TSV
  Warning: Proximal variant is not a missense mutation and will be skipped: chr7 67183212
  Warning: Proximal variant is not a missense mutation and will be skipped: chr9 98796327
  Warning: Proximal variant is not a missense mutation and will be skipped: chr12 62832210
  Warning: Proximal variant is not a missense mutation and will be skipped: chr16 21134327
  Warning: Proximal variant is not a missense mutation and will be skipped: chr16 21134336
  Completed
  Converting VCF to TSV
  Warning: Proximal variant is not a missense mutation and will be skipped: chr7 67183212
  Warning: Proximal variant is not a missense mutation and will be skipped: chr9 98796327
  Warning: Proximal variant is not a missense mutation and will be skipped: chr12 62832210
  Warning: Proximal variant is not a missense mutation and will be skipped: chr16 21134336
  Completed
  Generating Variant Peptide FASTA and Key File
  Completed
  Parsing the Variant Peptide FASTA and Key File
  Completed
  Calculating Manufacturability Metrics
  Completed
  Splitting TSV into smaller chunks
  Splitting TSV into smaller chunks - Entries 1-73
  Completed
  Generating Variant Peptide FASTA and Key Files
  Generating Variant Peptide FASTA and Key Files - Epitope Length 8 - Entries 1-146
  Generating Variant Peptide FASTA and Key Files - Epitope Length 9 - Entries 1-146
  Generating Variant Peptide FASTA and Key Files - Epitope Length 10 - Entries 1-146
  Generating Variant Peptide FASTA and Key Files - Epitope Length 11 - Entries 1-146
  Completed
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 8 with Method NetMHCpanEL - File MHC_Class_I/tmp/sample1_tumor.netmhcpan_el.HLA-C*14:02.8.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 9 with Method NetMHCpan - File MHC_Class_I/tmp/sample1_tumor.netmhcpan.HLA-C*14:02.9.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 9 with Method NetMHCpanEL - File MHC_Class_I/tmp/sample1_tumor.netmhcpan_el.HLA-C*14:02.9.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 10 with Method MHCflurry - File MHC_Class_I/tmp/sample1_tumor.MHCflurry.HLA-C*14:02.10.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 10 with Method NetMHCpan - File MHC_Class_I/tmp/sample1_tumor.netmhcpan.HLA-C*14:02.10.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 10 with Method NetMHCpanEL - File MHC_Class_I/tmp/sample1_tumor.netmhcpan_el.HLA-C*14:02.10.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 11 with Method MHCflurry - File MHC_Class_I/tmp/sample1_tumor.MHCflurry.HLA-C*14:02.11.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 11 with Method NetMHCpan - File MHC_Class_I/tmp/sample1_tumor.netmhcpan.HLA-C*14:02.11.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 8 with Method MHCflurry - File MHC_Class_I/tmp/sample1_tumor.MHCflurry.HLA-C*14:02.8.tsv_1-146
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 11 with Method NetMHCpanEL - File MHC_Class_I/tmp/sample1_tumor.netmhcpan_el.HLA-C*14:02.11.tsv_1-146
  Forcing tensorflow backend.
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 8 with Method MHCflurry - File MHC_Class_I/tmp/sample1_tumor.MHCflurry.HLA-C*14:02.8.tsv_1-146 - Completed
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 8 with Method NetMHCpan - File MHC_Class_I/tmp/sample1_tumor.netmhcpan.HLA-C*14:02.8.tsv_1-146
  Forcing tensorflow backend.
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 10 with Method MHCflurry - File MHC_Class_I/tmp/sample1_tumor.MHCflurry.HLA-C*14:02.10.tsv_1-146 - Completed
  Forcing tensorflow backend.
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 11 with Method MHCflurry - File MHC_Class_I/tmp/sample1_tumor.MHCflurry.HLA-C*14:02.11.tsv_1-146 - Completed

Command error:
      self.commandline_input(args)
    File "/opt/iedb/mhc_i/src/predict_binding.py", line 132, in commandline_input
      mhc_scores = mhc_predictor.predict(input.input_protein.as_amino_acid_text())
    File "/opt/iedb/mhc_i/src/seqpredictor.py", line 903, in predict
      scores.append(predictor.predict_sequence(sequence,pred))
    File "/opt/iedb/mhc_i/src/seqpredictor.py", line 305, in predict_sequence
      results = predict_netmhcpan(input_sequence_list, [(allele_name_or_sequence, self.length)])
    File "/opt/iedb/mhc_i/src/../method/netmhcpan-4.1-executable/netmhcpan_4_1_executable/__init__.py", line 258, in predict_many
      process = Popen(cmd, stdout=PIPE)
    File "/opt/conda/lib/python3.8/subprocess.py", line 854, in __init__
      self._execute_child(args, executable, preexec_fn, close_fds,
    File "/opt/conda/lib/python3.8/subprocess.py", line 1702, in _execute_child
      raise child_exception_type(errno_num, err_msg, err_filename)
  PermissionError: [Errno 13] Permission denied: '/opt/iedb/mhc_i/src/../method/netmhcpan-4.1-executable/netmhcpan_4_1_executable/netMHCpan'
  Forcing tensorflow backend.
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 10 with Method MHCflurry - File MHC_Class_I/tmp/sample1_tumor.MHCflurry.HLA-C*14:02.10.tsv_1-146 - Completed
  Forcing tensorflow backend.
  Making binding predictions on Allele HLA-C*14:02 and Epitope Length 11 with Method MHCflurry - File MHC_Class_I/tmp/sample1_tumor.MHCflurry.HLA-C*14:02.11.tsv_1-146 - Completed
  /opt/conda/lib/python3.8/site-packages/keras/src/engine/training_v1.py:2359: UserWarning: `Model.state_updates` will be removed in a future version. This property should not be used in TensorFlow 2.0, as `updates` are applied automatically.
    updates=self.state_updates,
  CRITICAL:pymp:An exception occured in thread 2: (, Command '['/opt/conda/bin/python', '/opt/iedb/mhc_i/src/predict_binding.py', 'netmhcpan', 'HLA-C*14:02', '9', 'MHC_Class_I/tmp/sample1_tumor.9.fa.split_1-146']' returned non-zero exit status 1.).
  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/pvactools/lib/pipeline.py", line 356, in call_iedb
      pvactools.lib.call_iedb.main(arguments)
    File "/opt/conda/lib/python3.8/site-packages/pvactools/lib/call_iedb.py", line 44, in main
      raise err
    File "/opt/conda/lib/python3.8/site-packages/pvactools/lib/call_iedb.py", line 39, in main
      (response_text, output_mode) = prediction_class_object.predict(args.input_file, args.allele, args.epitope_length, args.iedb_executable_path, args.iedb_retries, tmp_dir=args.tmp_dir)
    File "/opt/conda/lib/python3.8/site-packages/pvactools/lib/prediction_class.py", line 63, in predict
      response = run(arguments, stdout=response_fh, check=True)
    File "/opt/conda/lib/python3.8/subprocess.py", line 512, in run
      raise CalledProcessError(retcode, process.args,
  subprocess.CalledProcessError: Command '['/opt/conda/bin/python', '/opt/iedb/mhc_i/src/predict_binding.py', 'netmhcpan', 'HLA-C*14:02', '8', 'MHC_Class_I/tmp/sample1_tumor.8.fa.split_1-146']' returned non-zero exit status 1.
  
  During handling of the above exception, another exception occurred:
  
  Traceback (most recent call last):
    File "/opt/conda/bin/pvacseq", line 8, in 
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/pvactools/tools/pvacseq/main.py", line 123, in main
      args[0].func.main(args[1])
    File "/opt/conda/lib/python3.8/site-packages/pvactools/tools/pvacseq/run.py", line 138, in main
      pipeline.execute()
    File "/opt/conda/lib/python3.8/site-packages/pvactools/lib/pipeline.py", line 451, in execute
      self.call_iedb(chunks)
    File "/opt/conda/lib/python3.8/site-packages/pvactools/lib/pipeline.py", line 357, in call_iedb
      p.print("Making binding predictions on Allele %s and Epitope Length %s with Method %s - File %s - Completed" % (a, epl, method, filename))
    File "/opt/conda/lib/python3.8/site-packages/pymp/__init__.py", line 148, in __exit__
      raise exc_t(exc_val)
  TypeError: __init__() missing 1 required positional argument: 'cmd'

Work dir:
  /home/ubuntu/nextNEOpi/work/d8/182ae7593f5feda9fa6e7706bd4226

Tip: view the complete command output by changing to the process work dir and entering the command `cat .command.out`
```

**Comment by riederd** [comment]
Hi,
can you post the contents of `/home/ubuntu/nextNEOpi/work/d8/182ae7593f5feda9fa6e7706bd4226/.command.run`

**Comment by gri11** [comment]
Here it is the `.command.run` file when get the pVACseq error:
```
#!/bin/bash
# NEXTFLOW TASK: pVACseq (sample1)
set -e
set -u
NXF_DEBUG=${NXF_DEBUG:=0}; [[ $NXF_DEBUG > 1 ]] && set -x
NXF_ENTRY=${1:-nxf_main}

nxf_tree() {
    local pid=$1

    declare -a ALL_CHILDREN
    while read P PP;do
        ALL_CHILDREN[$PP]+=" $P"
    done < <(ps -e -o pid= -o ppid=)

    pstat() {
        local x_pid=$1
        local STATUS=$(2> /dev/null < /proc/$1/status egrep 'Vm|ctxt')

        if [ $? = 0 ]; then
        local  x_vsz=$(echo "$STATUS" | grep VmSize | awk '{print $2}' || echo -n '0')
        local  x_rss=$(echo "$STATUS" | grep VmRSS | awk '{print $2}' || echo -n '0')
        local x_peak=$(echo "$STATUS" | egrep 'VmPeak|VmHWM' | sed 's/^.*:\s*//' | sed 's/[\sa-zA-Z]*$//' | tr '\n' ' ' || echo -n '0 0')
        local x_pmem=$(awk -v rss=$x_rss -v mem_tot=$mem_tot 'BEGIN {printf "%.0f", rss/mem_tot*100*10}' || echo -n '0')
        local vol_ctxt=$(echo "$STATUS" | grep '\bvoluntary_ctxt_switches' | awk '{print $2}' || echo -n '0')
        local inv_ctxt=$(echo "$STATUS" | grep '\bnonvoluntary_ctxt_switches' | awk '{print $2}' || echo -n '0')
        cpu_stat[x_pid]="$x_pid $x_pmem $x_vsz $x_rss $x_peak $vol_ctxt $inv_ctxt"
        fi
    }

    pwalk() {
        pstat $1
        for i in ${ALL_CHILDREN[$1]:=}; do pwalk $i; done
    }

    pwalk $1
}

nxf_stat() {
    cpu_stat=()
    nxf_tree $1

    declare -a sum=(0 0 0 0 0 0 0 0)
    local pid
    local i
    for pid in "${!cpu_stat[@]}"; do
        local row=(${cpu_stat[pid]})
        [ $NXF_DEBUG = 1 ] && echo "++ stat mem=${row[*]}"
        for i in "${!row[@]}"; do
        if [ $i != 0 ]; then
            sum[i]=$((sum[i]+row[i]))
        fi
        done
    done

    [ $NXF_DEBUG = 1 ] && echo -e "++ stat SUM=${sum[*]}"

    for i in {1..7}; do
        if [ ${sum[i]} -lt ${cpu_peak[i]} ]; then
            sum[i]=${cpu_peak[i]}
        else
            cpu_peak[i]=${sum[i]}
        fi
    done

    [ $NXF_DEBUG = 1 ] && echo -e "++ stat PEAK=${sum[*]}\n"
    nxf_stat_ret=(${sum[*]})
}

nxf_mem_watch() {
    set -o pipefail
    local pid=$1
    local trace_file=.command.trace
    local count=0;
    declare -a cpu_stat=(0 0 0 0 0 0 0 0)
    declare -a cpu_peak=(0 0 0 0 0 0 0 0)
    local mem_tot=$(< /proc/meminfo grep MemTotal | awk '{print $2}')
    local timeout
    local DONE
    local STOP=''

    [ $NXF_DEBUG = 1 ] && nxf_sleep 0.2 && ps fx

    while true; do
        nxf_stat $pid
        if [ $count -lt 10 ]; then timeout=1;
        elif [ $count -lt 120 ]; then timeout=5;
        else timeout=30;
        fi
        read -t $timeout -r DONE || true
        [[ $DONE ]] && break
        if [ ! -e /proc/$pid ]; then
            [ ! $STOP ] && STOP=$(nxf_date)
            [ $(($(nxf_date)-STOP)) -gt 10000 ] && break
        fi
        count=$((count+1))
    done

    echo "%mem=${nxf_stat_ret[1]}"      >> $trace_file
    echo "vmem=${nxf_stat_ret[2]}"      >> $trace_file
    echo "rss=${nxf_stat_ret[3]}"       >> $trace_file
    echo "peak_vmem=${nxf_stat_ret[4]}" >> $trace_file
    echo "peak_rss=${nxf_stat_ret[5]}"  >> $trace_file
    echo "vol_ctxt=${nxf_stat_ret[6]}"  >> $trace_file
    echo "inv_ctxt=${nxf_stat_ret[7]}"  >> $trace_file
}

nxf_write_trace() {
    echo "nextflow.trace/v2"           > $trace_file
    echo "realtime=$wall_time"         >> $trace_file
    echo "%cpu=$ucpu"                  >> $trace_file
    echo "rchar=${io_stat1[0]}"        >> $trace_file
    echo "wchar=${io_stat1[1]}"        >> $trace_file
    echo "syscr=${io_stat1[2]}"        >> $trace_file
    echo "syscw=${io_stat1[3]}"        >> $trace_file
    echo "read_bytes=${io_stat1[4]}"   >> $trace_file
    echo "write_bytes=${io_stat1[5]}"  >> $trace_file
}

nxf_trace_mac() {
    local start_millis=$(nxf_date)

    /bin/bash -ue /home/ubuntu/nextNEOpi/work/ac/fe00fc945c7321719eac86bd8428b3/.command.sh

    local end_millis=$(nxf_date)
    local wall_time=$((end_millis-start_millis))
    local ucpu=''
    local io_stat1=('' '' '' '' '' '')
    nxf_write_trace
}

nxf_fd() {
    local FD=11
    while [ -e /proc/$$/fd/$FD ]; do FD=$((FD+1)); done
    echo $FD
}

nxf_trace_linux() {
    local pid=$$
    command -v ps &>/dev/null || { >&2 echo "Command 'ps' required by nextflow to collect task metrics cannot be found"; exit 1; }
    local num_cpus=$(< /proc/cpuinfo grep '^processor' -c)
    local tot_time0=$(grep '^cpu ' /proc/stat | awk '{sum=$2+$3+$4+$5+$6+$7+$8+$9; printf "%.0f",sum}')
    local cpu_time0=$(2> /dev/null < /proc/$pid/stat awk '{printf "%.0f", ($16+$17)*10 }' || echo -n 'X')
    local io_stat0=($(2> /dev/null < /proc/$pid/io sed 's/^.*:\s*//' | head -n 6 | tr '\n' ' ' || echo -n '0 0 0 0 0 0'))
    local start_millis=$(nxf_date)
    trap 'kill $mem_proc' ERR
    
    /bin/bash -ue /home/ubuntu/nextNEOpi/work/ac/fe00fc945c7321719eac86bd8428b3/.command.sh &
    local task=$!

    mem_fd=$(nxf_fd)
    eval "exec $mem_fd> >(nxf_mem_watch $task)"
    local mem_proc=$!

    wait $task

    local end_millis=$(nxf_date)
    local tot_time1=$(grep '^cpu ' /proc/stat | awk '{sum=$2+$3+$4+$5+$6+$7+$8+$9; printf "%.0f",sum}')
    local cpu_time1=$(2> /dev/null < /proc/$pid/stat awk '{printf "%.0f", ($16+$17)*10 }' || echo -n 'X')
    local ucpu=$(awk -v p1=$cpu_time1 -v p0=$cpu_time0 -v t1=$tot_time1 -v t0=$tot_time0 -v n=$num_cpus 'BEGIN { pct=(p1-p0)/(t1-t0)*100*n; printf("%.0f", pct>0 ? pct : 0) }' )

    local io_stat1=($(2> /dev/null < /proc/$pid/io sed 's/^.*:\s*//' | head -n 6 | tr '\n' ' ' || echo -n '0 0 0 0 0 0'))
    local i
    for i in {0..5}; do
        io_stat1[i]=$((io_stat1[i]-io_stat0[i]))
    done

    local wall_time=$((end_millis-start_millis))
    [ $NXF_DEBUG = 1 ] && echo "+++ STATS %CPU=$ucpu TIME=$wall_time I/O=${io_stat1[*]}"

    echo "nextflow.trace/v2"           > $trace_file
    echo "realtime=$wall_time"         >> $trace_file
    echo "%cpu=$ucpu"                  >> $trace_file
    echo "rchar=${io_stat1[0]}"        >> $trace_file
    echo "wchar=${io_stat1[1]}"        >> $trace_file
    echo "syscr=${io_stat1[2]}"        >> $trace_file
    echo "syscw=${io_stat1[3]}"        >> $trace_file
    echo "read_bytes=${io_stat1[4]}"   >> $trace_file
    echo "write_bytes=${io_stat1[5]}"  >> $trace_file

    [ -e /proc/$mem_proc ] && eval "echo 'DONE' >&$mem_fd" || true
    wait $mem_proc 2>/dev/null || true
    while [ -e /proc/$mem_proc ]; do nxf_sleep 0.1; done
}

nxf_trace() {
    local trace_file=.command.trace
    touch $trace_file
    if [[ $(uname) = Darwin ]]; then
        nxf_trace_mac
    else
        nxf_trace_linux
    fi
}
nxf_container_env() {
cat << EOF
export PATH="\$PATH:/home/ubuntu/nextNEOpi/bin"
EOF
}

nxf_sleep() {
  sleep $1 2>/dev/null || sleep 1;
}

nxf_date() {
    local ts=$(date +%s%3N);
    if [[ ${#ts} == 10 ]]; then echo ${ts}000
    elif [[ $ts == *%3N ]]; then echo ${ts/\%3N/000}
    elif [[ $ts == *3N ]]; then echo ${ts/3N/000}
    elif [[ ${#ts} == 13 ]]; then echo $ts
    else echo "Unexpected timestamp value: $ts"; exit 1
    fi
}

nxf_env() {
    echo '============= task environment ============='
    env | sort | sed "s/\(.*\)AWS\(.*\)=\(.\{6\}\).*/\1AWS\2=\3xxxxxxxxxxxxx/"
    echo '============= task output =================='
}

nxf_kill() {
    declare -a children
    while read P PP;do
        children[$PP]+=" $P"
    done < <(ps -e -o pid= -o ppid=)

    kill_all() {
        [[ $1 != $$ ]] && kill $1 2>/dev/null || true
        for i in ${children[$1]:=}; do kill_all $i; done
    }

    kill_all $1
}

nxf_mktemp() {
    local base=${1:-/tmp}
    mkdir -p "$base"
    if [[ $(uname) = Darwin ]]; then mktemp -d $base/nxf.XXXXXXXXXX
    else TMPDIR="$base" mktemp -d -t nxf.XXXXXXXXXX
    fi
}

nxf_fs_copy() {
  local source=$1
  local target=$2
  local basedir=$(dirname $1)
  mkdir -p $target/$basedir
  cp -fRL $source $target/$basedir
}

nxf_fs_move() {
  local source=$1
  local target=$2
  local basedir=$(dirname $1)
  mkdir -p $target/$basedir
  mv -f $source $target/$basedir
}

nxf_fs_rsync() {
  rsync -rRl $1 $2
}

on_exit() {
    exit_status=${nxf_main_ret:=$?}
    printf $exit_status > /home/ubuntu/nextNEOpi/work/ac/fe00fc945c7321719eac86bd8428b3/.exitcode
    set +u
    [[ "$tee1" ]] && kill $tee1 2>/dev/null
    [[ "$tee2" ]] && kill $tee2 2>/dev/null
    [[ "$ctmp" ]] && rm -rf $ctmp || true
    exit $exit_status
}

on_term() {
    set +e
    [[ "$pid" ]] && kill $pid 2>/dev/null
}

nxf_launch() {
    set +u; env - PATH="$PATH" ${TMP:+SINGULARITYENV_TMP="$TMP"} ${TMPDIR:+SINGULARITYENV_TMPDIR="$TMPDIR"} SINGULARITYENV_NXF_DEBUG=${NXF_DEBUG:=0} singularity exec -B /home/ubuntu/nextNEOpi --no-home -H /tmp -B /home/ubuntu/nextNEOpi/assets -B /tmp -B /home/ubuntu/nextNEOpi/resources -B /home/ubuntu/nextNEOpi/resources/databases/iedb:/opt/iedb -B /home/ubuntu/nextNEOpi/resources/databases/mhcflurry_data:/opt/mhcflurry_data /home/ubuntu/nextNEOpi/work/singularity/apps-01.i-med.ac.at-images-singularity-pVACtools_4.0.1_icbi_4ae2625d.sif /bin/bash -c "cd $PWD; eval $(nxf_container_env); /bin/bash /home/ubuntu/nextNEOpi/work/ac/fe00fc945c7321719eac86bd8428b3/.command.run nxf_trace"
}

nxf_stage() {
    true
    # stage input files
    rm -f sample1_vep_phased.vcf.gz
    rm -f sample1_vep_phased.vcf.gz.tbi
    rm -f sample1_hc_vep_pick.vcf.gz
    rm -f sample1_hc_vep_pick.vcf.gz.tbi
    rm -f .iedb_install_ok.chck
    ln -s /home/ubuntu/nextNEOpi/work/fe/e5e17b623757924536a9efc5219f68/sample1_vep_phased.vcf.gz sample1_vep_phased.vcf.gz
    ln -s /home/ubuntu/nextNEOpi/work/fe/e5e17b623757924536a9efc5219f68/sample1_vep_phased.vcf.gz.tbi sample1_vep_phased.vcf.gz.tbi
    ln -s /home/ubuntu/nextNEOpi/work/8f/53d82e08e1be2381a7518393da30c0/sample1_hc_vep_pick.vcf.gz sample1_hc_vep_pick.vcf.gz
    ln -s /home/ubuntu/nextNEOpi/work/8f/53d82e08e1be2381a7518393da30c0/sample1_hc_vep_pick.vcf.gz.tbi sample1_hc_vep_pick.vcf.gz.tbi
    ln -s /home/ubuntu/nextNEOpi/resources/databases/iedb/.iedb_install_ok.chck .iedb_install_ok.chck
}

nxf_unstage() {
    true
    [[ ${nxf_main_ret:=0} != 0 ]] && return
}

nxf_main() {
    trap on_exit EXIT
    trap on_term TERM INT USR2
    trap '' USR1

    [[ "${NXF_CHDIR:-}" ]] && cd "$NXF_CHDIR"
    export NXF_BOXID="nxf-$(dd bs=18 count=1 if=/dev/urandom 2>/dev/null | base64 | tr +/ 0A)"
    NXF_SCRATCH=''
    [[ $NXF_DEBUG > 0 ]] && nxf_env
    touch /home/ubuntu/nextNEOpi/work/ac/fe00fc945c7321719eac86bd8428b3/.command.begin
    set +u
    set -u
    [[ $NXF_SCRATCH ]] && cd $NXF_SCRATCH
    nxf_stage

    set +e
    ctmp=$(set +u; nxf_mktemp /dev/shm 2>/dev/null || nxf_mktemp $TMPDIR)
    local cout=$ctmp/.command.out; mkfifo $cout
    local cerr=$ctmp/.command.err; mkfifo $cerr
    tee .command.out < $cout &
    tee1=$!
    tee .command.err < $cerr >&2 &
    tee2=$!
    ( nxf_launch ) >$cout 2>$cerr &
    pid=$!
    wait $pid || nxf_main_ret=$?
    wait $tee1 $tee2
    nxf_unstage
}

$NXF_ENTRY
```

**Comment by riederd** [comment]
Hi, sorry for the late reply,

can you run the following commands:

```
singularity exec -B /home/ubuntu/nextNEOpi --no-home -H /tmp -B /home/ubuntu/nextNEOpi/assets -B /tmp -B /home/ubuntu/nextNEOpi/resources -B /home/ubuntu/nextNEOpi/resources/databases/iedb:/opt/iedb -B /home/ubuntu/nextNEOpi/resources/databases/mhcflurry_data:/opt/mhcflurry_data /home/ubuntu/nextNEOpi/work/singularity/apps-01.i-med.ac.at-images-singularity-pVACtools_4.0.1_icbi_4ae2625d.sif /bin/bash 
ls -la /opt/iedb/mhc_i/src/../method/netmhcpan-4.1-executable/netmhcpan_4_1_executable/netMHCpan
```

and post the output?

**Comment by riederd** [comment]
Should be fixed now in v1.4.1, please reopen if not.
